src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -335,42 +331,24 @@ export function ChangeDataView({
     if (textBasedLanguages?.length) {
       panelItems.push(
         <EuiHorizontalRule margin="none" key="textbasedLanguages-divider" />,
-        <EuiFlexGroup
-          alignItems="center"
-          gutterSize="none"
-          justifyContent="spaceBetween"
-          data-test-subj="select-text-based-language-panel"
-          key="text-based-languages-switcher"
-          css={css`
-            margin: ${euiTheme.size.s};
-            margin-bottom: 0;
-          `}
-        >
-          <EuiFlexItem grow={false}>
-            <EuiText size="s">
-              <h5>
-                {i18n.translate(
-                  'unifiedSearch.query.queryBar.indexPattern.textBasedLanguagesLabel',
-                  {
-                    defaultMessage: 'Text-based query languages',
-                  }
-                )}
-              </h5>
-            </EuiText>
-          </EuiFlexItem>
-        </EuiFlexGroup>,
-        <TextBasedLanguagesList
-          key="text-based-languages-list"
-          textBasedLanguages={textBasedLanguages}
-          selectedOption={triggerLabel}
-          onChange={(lang) => {
-            setTriggerLabel(lang);
-            setPopoverIsOpen(false);
-            setIsTextBasedLangSelected(true);
-            // also update the query with the sql query
-            onTextLangQuerySubmit?.({ sql: `SELECT * FROM "${trigger.title}"` });
-          }}
-        />
+        <EuiPanel color="transparent" paddingSize="none">
+          <EuiButton
+            color="success"
+            size="s"
+            fullWidth
+            onClick={() => onTextBasedSubmit({ esql: `from ${trigger.title} | limit 10` })}
+            data-test-subj="select-text-based-language-panel"
+          >
+            {i18n.translate('unifiedSearch.query.queryBar.textBasedLanguagesTryLabel', {
+              defaultMessage: 'Try ES | QL',
```

Comment:
```suggestion
              defaultMessage: 'Try ES|QL',
```
Should there be spaces here?

Reply:
No without spaces is the correct here, you are right 👍